### PR TITLE
feat(console): health-check - use new cron-form component

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue

n/a

🙏  Need to merge this one before  https://github.com/gravitee-io/gravitee-ui-particles/pull/300

## Description

Use new cron-form component
image

## Additional context

🏗️ Your changes can be tested here and will be available soon:
      Console: [https://pr.team-apim.gravitee.dev/5482/console](https://pr.team-apim.gravitee.dev/5482/console)
      Portal: [https://pr.team-apim.gravitee.dev/5482/portal](https://pr.team-apim.gravitee.dev/5482/portal)
      Management-api: [https://pr.team-apim.gravitee.dev/5482/api/management](https://pr.team-apim.gravitee.dev/5482/api/management)
      Gateway v4: [https://pr.team-apim.gravitee.dev/5482](https://pr.team-apim.gravitee.dev/5482)
      Gateway v3: [https://pr.gateway-v3.team-apim.gravitee.dev/5482](https://pr.gateway-v3.team-apim.gravitee.dev/5482)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-fdfpfwlmhm.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "@danielmoncada/angular-datetime-picker": "^16.0.1",
     "@danielmoncada/angular-datetime-picker-moment-adapter": "^3.0.1",
     "@fontsource/libre-franklin": "4.4.5",
-    "@gravitee/ui-analytics": "7.30.0",
+    "@gravitee/ui-analytics": "7.33.0",
```
Comment (phiz71) [inline gravitee-apim-console-webui/package.json]:
```suggestion
    "@gravitee/ui-analytics": "7.35.0",
```

File: gravitee-apim-console-webui/package.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
...
     "@gravitee/ui-components": "3.44.0",
-    "@gravitee/ui-particles-angular": "7.30.0",
-    "@gravitee/ui-policy-studio-angular": "7.30.0",
+    "@gravitee/ui-particles-angular": "^7.34.0-form-cron-p2-b51c899",
+    "@gravitee/ui-policy-studio-angular": "7.33.0",
```
Comment (phiz71) [inline gravitee-apim-console-webui/package.json]:
```suggestion
    "@gravitee/ui-particles-angular": "^7.35.0",
    "@gravitee/ui-policy-studio-angular": "7.35.0",
```

**Review by phiz71 — CHANGES_REQUESTED** [review]
Just need to update the ui-particles version

File: gravitee-apim-console-webui/package.json (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/package.json]: haha yes of course, i've juste merge ui-p yesterday.

**Review by phiz71 — APPROVED** [review]

**Pull request merged**